docs/source/query-syntax.rst

```diff
@@ -0,0 +1,133 @@
+.. _query-syntax:
+
+Search
+======
+
+The MLflow UI and API support searching runs within a single experiment or a group of experiments
+using a search filter API. This API is a simplified version of the SQL ``WHERE`` clause.
+
+.. contents:: Table of Contents
+  :local:
+  :depth: 3
+
+Syntax
+------------------
```

Comment:
Is it ok to have thus underline longer than the sub-title string? I know sphinx does not like some stuff like this

Reply:
Longer is OK. It's shorter that breaks.